modules/installing-olm-using-ansible.adoc

```diff
@@ -0,0 +1,59 @@
+// Module included in the following assemblies:
+//
+// * operators/installing-operator-framework.adoc
+
+[id='installing-olm-using-ansible_{context}']
+= Installing Operator Lifecycle Manager using Ansible
+ifdef::openshift-enterprise[]
+:pb-prefix: /usr/share/ansible/openshift-ansible/
+endif::[]
+ifdef::openshift-origin[]
+:pb-prefix: ~/openshift-ansible/
+endif::[]
+
+Installing the Technology Preview Operator Framework is handled via an Operator. To install it, use the included playbook with the {product-title} `openshift-ansible` installer after installing your cluster.
+
+[NOTE]
+====
+Alternatively, the Technology Preview Operator Framework can be  installed
+during initial cluster installation. See
+xref:../install/configuring_inventory_file.adoc#install-config-configuring-inventory-file[Configuring Your Inventory File] for separate instructions.
+====
+
+.Prerequisites
+
+- An existing {product-title} 3.11 cluster
+- Access to the cluster using an account with `cluster-admin` permissions
```

Comment:
No, this doc is only about installing it, which is done by admins. I agree that having a general usage doc is helpful.

Reply:
Thanks! @robszumski There is the description section about the usage of the OLM in this doc too. But, because of the usage for the general users is not very clear in 3.11, I suggest that we highlight the point which the operations of the OLM(including create operators via the OLM actions, etc...) only for the `cluster-admin` users. What do you think?

Reply:
@jianzhangbjz @robszumski @ecordell How about a [NOTE] box in "Launching Your First Operator" that says something like:

"During this Technology Preview phase, focus is placed on cluster administrator usage. The following procedure can currently only be run by a user with cluster-admin permissions, however in upcoming releases, developer users with common permissions will be able to perform certain OLM actions themselves."

Also added a truncated version to "What's in the Technology Preview?" earlier on:

"During this Technology Preview phase, focus is placed on cluster administrator usage. In upcoming releases, developers users with common permissions will be able to perform certain OLM actions themselves."

Better way to say this?

Reply:
We have the functionality for folks to delegate access today. Various sentences that can be strung together however you'd like:

"Installing an Operator within a namespace is done through a Subscription object, which the cluster administrator can create and manage across the entire cluster. Users with a Role that includes access to Subscriptions (eg, `admin` role applied within a Project) will also be able to manage subscriptions for their project."

"Within a Project, all users with the `edit` role will be able to create, manage and delete application instances managed by Operators that have been installed in the Project, in a self-service manner, just like a cloud service."

"Further granularity can be established with a customized Role to grant or prevent users the ability to manage certain Operator instance types."

Reply:
>  Users with a Role that includes access to Subscriptions (eg, admin role applied within a Project) will also be able to manage subscriptions for their project."

@robszumski Yes, you're right. But, this user cannot access the `CatalogSource` to init their `Subscriptions`. They will get the below errors unless the `cluster-admin` users grant the permission: `# oc policy add-role-to-user view jiazha2 -n operator-lifecycle-manager`
```
catalogsources.operators.coreos.com is forbidden: User "jiazha2" cannot list catalogsources.operators.coreos.com in the namespace "operator-lifecycle-manager": no RBAC policy matched
``` 
And, they cannot create the Cluster against their Operator either since no approprivate permission.  For example, if they want to create the etcd-cluster, they need to get the permission first:
```
# oc policy add-role-to-user etcdoperator.v0.9.2-role-9rswg jiazha2 --role-namespace=jiazha
```